lib/ansible_tower_client/valid_json.rb

```diff
@@ -0,0 +1,12 @@
+module AnsibleTowerClient
+  module ValidJSON
+    def validate(*args)
+      begin
+        JSON.parse(args.first)
```

Comment:
Why `*args` above and `args.first` here?  Isn't it always a single string being passed in?